weldx/asdf/schemas/weldx.bam.de/weldx/core/transformations/coordinate_system_hierarchy-1.0.0.yaml

```diff
@@ -11,10 +11,24 @@ description: |
   is a tree.
 type: object
 properties:
+  name:
+    type: string
+
   root_system_name:
     description: |
       Name of the root coordinate system
     type: string
+
+  subsystems:
+    type: array
+    items:
+      type: string
+
+  subsystem_data:
```

Comment:
```suggestion
  subsystems:
```

Reply:
could we not use a single list? (list of names + named subsystems seems redundant)
Maybe with `wx_property_tag` validator?

Reply:
The list with the names contains only direct child subsystems. Nested ones are excluded. I added a corresponding description to clarify that.